src/components/VideoDetailCard/VideoDetailCard.component.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardContent from '@material-ui/core/CardContent';
+import Collapse from '@material-ui/core/Collapse';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import ReactMarkdown from 'react-markdown';
+
+import useStyles from './VideoCardDetail.styles';
+
+function urlify(text, maxLength = 'full') {
+  const urlRegex = /(https?:\/\/[^\s]+)/g;
+  return text.replace(urlRegex, (url) => {
+    return `[${
+      maxLength === 'full' || !maxLength ? url : `${url.substring(0, maxLength)}...`
+    }](#${url})`;
+  });
+}
```

Comment:
URLs are not generated correctly, please take a look:

![Screen Shot 2020-12-22 at 17 22 14](https://user-images.githubusercontent.com/3632628/102942234-6c621c00-447a-11eb-8bb9-3e708a7c2236.png)